<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ASD-smirnova" ignored="0" name="Surefire test" tests="18" failures="1" timestamp="12 апр 2018 11:56:11 GMT" time="0.049" errors="0">
  <testcase name="test2" time="0.006" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="checkThreads" time="0.0" classname="company.SmokeTest"/>
  <testcase name="test1" time="0.0" classname="company.SmokeTest">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at company.SmokeTest.test1(SmokeTest.java:54)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="method1" time="0.0" classname="company.SmokeTest"/>
  <testcase name="method2" time="0.0" classname="company.SmokeTest"/>
  <testcase name="method3" time="0.0" classname="company.SmokeTest"/>
  <testcase name="errorMethod" time="0.0" classname="company.SmokeTest"/>
  <testcase name="dependOnMethod1" time="0.001" classname="company.SmokeTest"/>
  <testcase name="dependsOnGroup1" time="0.0" classname="company.SmokeTest"/>
</testsuite> <!-- Surefire test -->
